import-service/package.json

```diff
@@ -0,0 +1,44 @@
+{
+  "name": "import-service",
+  "version": "1.0.0",
+  "description": "Serverless webpack example using Typescript",
+  "main": "handler.js",
+  "scripts": {
+    "test": "jest",
+    "dev": "sls offline",
+    "deploy:prod": "serverless deploy --region eu-west-1 --stage prod",
+    "deploy:dev": "serverless deploy --region eu-west-1 --stage dev"
+  },
+  "dependencies": {
+    "aws-sdk": "^2.792.0",
```

Comment:
You can reduce your lambda bundle size with moving `aws-sdk` to `devDependencies`